card/mod/02_basic_types/set/type/pointer.rb

```diff
@@ -211,8 +211,11 @@ def wrap_item item, args
   end
 end
 
-
-event :standardize_items, :before=>:approve, :on=>:save do
+# while a card's card type and content are updated in the same request, 
+# the new module will override the old module's events and functions.
+# this event is only on pointer card. Other type cards do not have this event,
+# so it is not overridden and will be run while updating type and content in the same request. 
+event :standardize_items, :before=>:approve, :on=>:save, :when=>proc{  |c| c.type_id == Card::PointerID  } do
```

Comment:
Yeah, that will be pretty important in making sure the better way to fix it actually works.